Makefile

```diff
@@ -139,9 +139,19 @@ uninstall:
 	rm -f $(DESTDIR)/$(BINDIR)/buildah
 	rm -f $(PREFIX)/share/man/man1/buildah*.1
 	rm -f $(DESTDIR)/$(BASHINSTALLDIR)/buildah
+	rm -f $(DESTDIR)/$(BASHINSTALLDIR)/buildah
+
+contrib/completions/bash/buildah: ./bin/buildah
+	./bin/buildah completion bash > $@
+contrib/completions/fish/buildah: ./bin/buildah
+	./bin/buildah completion fish > $@
+contrib/completions/powershell/buildah: ./bin/buildah
+	./bin/buildah completion powershell > $@
+contrib/completions/zsh/buildah: ./bin/buildah
+	./bin/buildah completion zsh > $@
 
 .PHONY: install.completions
-install.completions:
+install.completions: contrib/completions/bash/buildah
```

Comment:
this should install for zsh and fish as well, see the podman Makefile for an example

Reply:
Adding.